src/main/java/com/bootcamp/demo/models/User.java

```diff
@@ -0,0 +1,72 @@
+package com.bootcamp.demo.models;
```

Comment:
To be honest I haven't given it much thought at the time. I thought that it would contain more classes that are different types that are all models (homogeneous contents).

Reply:
Every model are different usually. So its not homogeneous package. :)

Reply:
@dbernatt  you could also include a URL to a more detailed explanation if you think it might a more detailed overview/explanation might help.

Reply:
Here is the explanation : [link](https://github.com/nymphadora-tonks-team/booking-jb/pull/16#discussion_r748357408)

Reply:
Thanks. I understand the difference now.